docs/product-assembly_pam-on-openshift/src/main/asciidoc/apb-deploy-proc.adoc

```diff
@@ -0,0 +1,35 @@
+[id='apb-deploy-proc']
+= Deploying a {PRODUCT} environment using the Automation Broker
+
+To deploy a {PRODUCT} environment using the Automation Broker, you must find the Ansible Playbook in the OpenShift catalog, run it, and set the parameters as required. 
+
+.Procedure
+
+. In the OpenShift Web UI, select *Add to Project -> Browse Catalog*.
+
+. In the search field, type {PRODUCT}.
+
+. Select the _{PRODUCT} {PRODUCT_VERSION} APB_ catalog entry.
```

Comment:
"Select the **{PRODUCT} {PRODUCT_VERSION} APB** catalog entry" (also bold - all referenced UI elements in bold, not ital)